### PR TITLE
support Raspbian armhf repos

### PR DESCRIPTION
**Pull request by gianmarco-mameli**
add support for configure Raspbian armhf repos

### CODE REVIEW

**Comment by gianmarco-mameli** [comment]
Hi, I've completed the procedure to sign the CLA, thanks

**Review by mkayontour — CHANGES_REQUESTED** [review]
Thanks for the PR.
I don't have a raspberry at the moment to check which variables are available but, it think it is possible to include raspbian in the first lookup. 
`ansible_os_family` should be **debian** i think, afterwards you can create a file in `vars/` named, 
`Debian-Raspbian.yml`.

File: roles/repos/tasks/main.yml [hunk 1]
```diff
@@ -28,5 +28,19 @@
...
+    params:
+      files:
+        - "{{ ansible_lsb.id }}-{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
+        - "{{ ansible_lsb.id }}-{{ ansible_distribution_major_version }}.yml"
+        - "{{ ansible_lsb.id }}-{{ ansible_distribution }}.yml"
+        - "{{ ansible_lsb.id }}.yml"
+        - default.yml
+      paths:
+        - "{{ role_path }}/vars"
+
```
[the remark this answers is not in the document]
Comment (gianmarco-mameli) [reply roles/repos/tasks/main.yml:44]: Hi, you're right, as soon as the Debian.yml doesn't exist the last variable file imported il Debian-Raspbian.yml overriding correctly the repos. I'm going to commit the changes. Thanks

**Comment by gianmarco-mameli** [comment]
I don't know why the CLA is failing

**Comment by martialblog** [comment]
@bobapple  to the rescue

**Comment by bobapple** [comment]
@cla-bot check

**Comment by gianmarco-mameli** [comment]
Thanks

**Comment by mkayontour** [comment]
Thanks again for your contribution!

**Pull request merged**